test/integration/features/basic.feature

```diff
@@ -0,0 +1,21 @@
+# 
+Feature: Bringup
+  In order to use MiniShift
+  As a user
+  I need to be able to bring up a test environment
+
+
+  Scenario: Docker environment variables
+    Given a "developer"
+     When executing "minishift start --vm-driver=vbox --docker-env=FOO=BAR --docker-env=BAZ=BAT"
```

Comment:
currently failing due to #434, but perhaps better to use kvm, as centos-ci relies on this.

We can improve later to allowing several scenarios